src/Examples.Console/Program.cs

```diff
@@ -42,12 +41,13 @@ internal class DynatraceOptions
         [Option('d', "duration", Default = 2, HelpText = "Total duration in minutes to run the demo. Run atleast for a min to see metrics flowing.", Required = false)]
         public int DurationInMins { get; set; }
 
-        [Option('u', "url", Default = "http://127.0.0.1:14499/metrics/ingest", HelpText = "Dynatrace metrics ingest API URL.", Required = false)]
+        [Option('u', "url", Default = "http://localhost:14499/metrics/ingest", HelpText = "Dynatrace metrics ingest API URL.", Required = false)]
```

Comment:
Isn't this already the default of the options class?

Reply:
yes, but this is the default for the command line option in the example package.

Reply:
Could we just not set anything if not specified so it takes the options class' default?

Reply:
Yes, the advantage of setting it here as well is when using the command line flag `--help` on the example application it will print that information, which might be helpful.

Reply:
Could also be the opposite as it might make users think they have to supply this value in their app as well then since this is what the example app is doing too. I'd rather remove it but this can also be done in your follow-up PR.